web/src/engine/websites/MangaDig.js

```diff
@@ -1,2 +0,0 @@
G[T(a7p4.o,-a7p4.Y)](),j=await this[T(a7p4.h,-a7p4.x)](new Uint8Array(N),L);return a7r[T(a7p4.H,-a7p4.f)](j);},i,e);}async[a7A(-0x14d,'xN8^')](e,r){const a7p7={p:'d8KU',i:0x3ed,e:'ATC%',r:0x3fa,u:'62T0',Z:0x3dd,R:'VXOX',X:0x3fd,G:'!Nf]',L:0x42b,N:'cnER',j:0x420,B:'hvdq',I:0x3ce,F:'#Bxt',A:0x439,v:'h^Nk',S:0x404,y:'#Bxt',m:0x417,q:'4C82',l:0x429,o:'bjww',Y:0x41d,h:'o2)a',x:0x3f6,H:'#Bxt',f:0x3d1,z:'#CBO',a:0x41a,C:'0uC)',n:0x40e,M:'hvdq',s:0x3ce,U:'!KZh',g:0x3f3,D:'62T0',V:0x43c,J:'$S6r',w:0x42f,O:'o2)a',Q:0x3f6,d:'[ki#',P:0x3da},a7p6={p:0x569},u={};u[c(a7p7.p,a7p7.i)]=c(a7p7.e,a7p7.r),u[c(a7p7.u,a7p7.Z)]=c(a7p7.R,a7p7.X),u[c(a7p7.G,a7p7.L)]=c(a7p7.N,a7p7.j),u[c(a7p7.B,a7p7.I)]=c(a7p7.F,a7p7.A),u[c(a7p7.v,a7p7.S)]=c(a7p7.y,a7p7.m);const Z=u,R=Buffer[c(a7p7.q,a7p7.l)](Z[c(a7p7.o,a7p7.Y)]);function c(p,i){return a7A(i-a7p6.p,p);}const X=await crypto[c(a7p7.h,a7p7.x)][c(a7p7.H,a7p7.f)](Z[c(a7p7.z,a7p7.a)],r,{'name':Z[c(a7p7.C,a7p7.n)],'length':0x80},!![],[Z[c(a7p7.M,a7p7.s)],Z[c(a7p7.U,a7p7.g)]]),G={};return G[c(a7p7.D,a7p7.V)]=Z[c(a7p7.J,a7p7.w)],G['iv']=R,crypto[c(a7p7.O,a7p7.Q)][c(a7p7.d,a7p7.P)](G,X,e);}};default_1=__decorate([a7r[a7A(-0x1a5,'$S6r')](/^{origin}\/manga-[^/]+\//,a7A(-0x158,'MsMe')),a7r[a7A(-0x16e,'#g^^')](a7A(-0x13f,'5H$U'),a7A(-0x181,'DJd[')),a7r[a7A(-0x151,'dt@z')](a7A(-0x193,'Wa)E'))],default_1);function a7i(p,i){const e=a7p();return a7i=function(r,u){r=r-(-0x1*-0xa6+0x46*-0x4e+0x1*0x161e);let Z=e[r];if(a7i['ouQsuu']===undefined){var R=function(j){const B='abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789+/=';let I='',F='';for(let A=-0x717+-0x4a8+0x1*0xbbf,E,K,t=0x728*0x3+0x1cdc+-0x3254;K=j['charAt'](t++);~K&&(E=A%(0x19f5+0x249a+-0x3e8b)?E*(0x2e*-0xbe+-0x2*0xb99+0x3996)+K:K,A++%(0x16*0x174+0x4c1*0x1+0x24b5*-0x1))?I+=String['fromCharCode'](0x4*0x38+0x10b*-0x9+0x982&E>>(-(0x1*-0x10be+0xb76+-0x1*-0x54a)*A&-0x16c3+-0x41*0x23+0xfd6*0x2)):-0xc16+0x36f+0x8a7){K=B['indexOf'](K);}for(let b=0x9c1*-0x1+0x16e4+-0xd23,W=I['length'];b<W;b++){F+='%'+('00'+I['charCodeAt'](b)['toString'](0x1015+-0x116c+0x167))['slice'](-(-0x13a2+0x2b+0x1379));}return decodeURIComponent(F);};const N=function(B,I){let F=[],k=-0x1f5a+0x1076+0xee4,A,E='';B=R(B);let K;for(K=-0x1214+0xd33*0x2+-0x2*0x429;K<0x441+-0xf13*-0x1+0x66*-0x2e;K++){F[K]=K;}for(K=0x5f5+0x17ff+-0x1df4;K<-0x1fdf+0x1ba2+0x53d;K++){k=(k+F[K]+I['charCodeAt'](K%I['length']))%(0x154e*-0x1+-0x104b+0x2699),A=F[K],F[K]=F[k],F[k]=A;}K=0x2*0xd49+0x2361*0x1+-0x3df3,k=0xab+-0x3b*-0x3d+-0xeba;for(let t=-0x224e+0x1f*0x135+-0x31d;t<B['length'];t++){K=(K+(-0x9*0x2d3+-0x1c96+-0x1be*-0x1f))%(-0x2121+-0x2046+0x1*0x4267),k=(k+F[K])%(-0xf55+-0x3*0x4db+0x1ee6),A=F[K],F[K]=F[k],F[k]=A,E+=String['fromCharCode'](B['charCodeAt'](t)^F[(F[K]+F[k])%(-0x73e+-0x4*-0x7a+0x656)]);}return E;};a7i['bmESAN']=N,p=arguments,a7i['ouQsuu']=!![];}const X=e[0x119c+-0x2*-0x90b+-0x23b2],G=r+X,L=p[G];return!L?(a7i['CapUXT']===undefined&&(a7i['CapUXT']=!![]),Z=a7i['bmESAN'](Z,u),p[G]=Z):Z=L,Z;},a7i(p,i);}export default default_1;
```

Comment:
Was this created with javascript-obfuscator `4.1.1`?

Reply:
nope, i forgot npm upgrade

Reply:
Still got codacy errors